{
  "uid" : "33259b2f71a2e123",
  "name" : "deployer has the default admin role",
  "fullName" : "Contract: ERC20PresetMinterPauser.deployer has the default admin role",
  "historyId" : "956ac89c93ae71794800db2ab4c8fd09",
  "time" : {
    "start" : 1635782729975,
    "stop" : 1635782730937,
    "duration" : 962
  },
  "description" : "",
  "descriptionHtml" : "",
  "status" : "broken",
  "statusMessage" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a04375b0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
  "statusTrace" : "Error: Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a04375b0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))\n                    at Context.<anonymous> (test/token/ERC20/presets/ERC20PresetMinterPauser.test.js:21:51)\n                    at processImmediate (node:internal/timers:464:21)",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "suite",
    "value" : "Contract: ERC20PresetMinterPauser"
  }, {
    "name" : "testClass",
    "value" : "Contract: ERC20PresetMinterPauser"
  }, {
    "name" : "testMethod",
    "value" : "deployer has the default admin role"
  }, {
    "name" : "package",
    "value" : "Contract: ERC20PresetMinterPauser"
  }, {
    "name" : "resultFormat",
    "value" : "allure1"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ {
      "uid" : "25fc1b3dabe0751e",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f238be9ea60>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782608441,
        "stop" : 1635782609045,
        "duration" : 604
      }
    }, {
      "uid" : "663b49ef4c883d75",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f238be9ef40>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782607286,
        "stop" : 1635782607847,
        "duration" : 561
      }
    }, {
      "uid" : "353eac56a112ab5f",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a17de880>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782606836,
        "stop" : 1635782607401,
        "duration" : 565
      }
    }, {
      "uid" : "ff2c632bf37a55e7",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0689eb0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782583568,
        "stop" : 1635782584534,
        "duration" : 966
      }
    }, {
      "uid" : "e3bfe825c4b5dff0",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0fbc5b0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782580261,
        "stop" : 1635782580821,
        "duration" : 560
      }
    }, {
      "uid" : "771dcf9a9064b609",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0441ac0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782527935,
        "stop" : 1635782528497,
        "duration" : 562
      }
    }, {
      "uid" : "fe7020d6b7164c91",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a28750d0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782516040,
        "stop" : 1635782516599,
        "duration" : 559
      }
    }, {
      "uid" : "ab4faadb8628fee4",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a04338b0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782513568,
        "stop" : 1635782514128,
        "duration" : 560
      }
    } ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 4,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 4
      },
      "items" : [ {
        "uid" : "865782acad33bf7a",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/8//#testresult/865782acad33bf7a",
        "status" : "broken",
        "statusDetails" : "Returned error: Transaction simulation failed: Error processing Instruction 0: invalid account data for instruction",
        "time" : {
          "start" : 1635590301245,
          "stop" : 1635590316933,
          "duration" : 15688
        }
      }, {
        "uid" : "9741b4b47634cb86",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/7//#testresult/9741b4b47634cb86",
        "status" : "broken",
        "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a067ec40>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
        "time" : {
          "start" : 1635555004420,
          "stop" : 1635555005412,
          "duration" : 992
        }
      }, {
        "uid" : "1f22b9dc27152b87",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/6//#testresult/1f22b9dc27152b87",
        "status" : "broken",
        "statusDetails" : "HH110: Invalid JSON-RPC response received: <html>\n                <head><title>502 Bad Gateway</title></head>\n                <body>\n                <center><h1>502 Bad Gateway</h1></center>\n                <hr><center>nginx/1.18.0 (Ubuntu)</center>\n                </body>\n                </html>\n                ",
        "time" : {
          "start" : 1635551255531,
          "stop" : 1635551255616,
          "duration" : 85
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "33259b2f71a2e123.json",
  "parameterValues" : [ ]
}